src/Qir/qir-utils.ps1

```diff
@@ -192,9 +192,10 @@ function Build-CMakeProject {
         $env:CXX = "clang++.exe"
         $env:RC = "clang++.exe"
 
-        if (!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") {
+        if ((!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") -or `
+            (Test-Path Env:AGENT_OS)) {
```

Comment:
In one file the fragment is 
`Env:/AGENT_OS`    (with `/`)
in the other - 
`Env:AGENT_OS`      (without `/`).

It probably makes sense to use `Env:/AGENT_OS` in both.